sdk/storage/tests.yml

```diff
@@ -35,5 +35,5 @@ jobs:
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
-        VERSIONED_STORAGE_ACCOUNT_NAME: $(azure-sdk-test-versioned-storage-account-name)
-        VERSIONED_STORAGE_ACCOUNT_KEY: $(azure-sdk-test-versioned-storage-account-key)
+        VERSIONED_STORAGE_ACCOUNT_NAME: $(java-storage-versioned-account-name)
```

Comment:
Changed the name of the secret?

Reply:
Yeah. I forgot to merge name change in other PR. This isn't yet used in pipelines.